block/manager.go

```diff
@@ -163,11 +157,14 @@ func (m *Manager) Start(ctx context.Context) error {
 	if isSequencer {
 		// Sequencer must wait till DA is synced to start submitting blobs
 		<-m.DAClient.Synced()
+
+		bytesProducedC := make(chan int, 1)
```

Comment:
why size 1? it can lead to health toggling in some race conditions

Reply:
Hmm, I will change to make it cleaner, but what specific problem do you see? Health toggling can occur regardless

Reply:
on some cases, u might produce 2 blocks while the submit loop as a consumer won't read it fast enough for some reason.